src/main/webapp/js/index.js

```diff
@@ -67,27 +85,37 @@ function createReceiptCardElement(receipt) {
   }
 
   receiptCardClone.querySelector('img').src = receipt.imageUrl;
+  receiptCardClone.querySelector('.col-md-6').id = receipt.id;
```

Comment:
nit: can we use id's for these, just seems a little brittle to rely on css as I can see the css as  being something that could change later.

Reply:
Could you clarify what you mean by this? Like change col-md-6?